packages/alea-frontend/components/QuizDashboard.tsx

```diff
@@ -62,17 +62,49 @@ const QuizDurationInfo = ({ quizStartTs, quizEndTs, feedbackReleaseTs }) => {
         marginTop: '5px',
       }}
     >
-      <p style={{ color: '#1e4620' }}>
+      <Typography sx={{ color: '#1e4620' }}>
         Quiz is <strong>{`${quizDuration} minutes`}</strong> long, and it will take additional{' '}
         <strong>{`${feedbackDuration} minutes`}</strong> for feedback release
-      </p>
+      </Typography>
     </Box>
   );
 };
+
 interface QuizDashboardProps {
   courseId: string;
 }
 
+function validateQuizUpdate(
+  originalProblems: Record<string, FTMLProblemWithSolution>,
+  newProblems: Record<string, FTMLProblemWithSolution>,
+  totalStudents: number
+) {
+  if (totalStudents > 0) {
```

Comment:
use early return pattern to reduce nesting. This helps improve readability and maintainability of the code
```suggestion
  if (totalStudents === 0) return { valid: true};
```
https://medium.com/softwarecraft-mastery/the-return-early-pattern-c554f47fe58a